community/kernel/src/main/java/org/neo4j/kernel/api/index/IndexAccessor.java

```diff
@@ -90,6 +91,17 @@
      */
     ResourceIterator<File> snapshotFiles() throws IOException;
 
+    /**
+     * Verifies that each value in this index is unique.
+     * Index is guaranteed to not change while this call executes.
+     *
+     * @param propertyAccessor {@link PropertyAccessor} for accessing properties from database storage
+     * in the event of conflicting values.
+     * @throws IndexEntryConflictException for first detected uniqueness conflict, if any.
+     * @throws IOException on error reading from source files.
+     */
+    void verifyDeferredConstraints( PropertyAccessor propertyAccessor ) throws IndexEntryConflictException, IOException;
```

Comment:
Oh, that's right. It just so happens to be correct since this method is only called when creating a uniqueness constraint. I'll put this into the uniqueness variant only.